jenkins/rootfs-trigger.jpl

```diff
@@ -0,0 +1,147 @@
+#!/usr/bin/env groovy
+
+/*
+  Copyright (C) 2020 Collabora Limited
+  Author: Lakshmipathi Ganapathi <lakshmipathi.ganapathi@collabora.com>
+
+  This module is free software; you can redistribute it and/or modify it under
+  the terms of the GNU Lesser General Public License as published by the Free
+  Software Foundation; either version 2.1 of the License, or (at your option)
+  any later version.
+
+  This library is distributed in the hope that it will be useful, but WITHOUT
+  ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+  FOR A PARTICULAR PURPOSE.  See the GNU Lesser General Public License for more
+  details.
+
+  You should have received a copy of the GNU Lesser General Public License
+  along with this library; if not, write to the Free Software Foundation, Inc.,
+  51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA
+*/
+
+/* ----------------------------------------------------------------------------
+ * Jenkins parameters
+
+KCI_CORE_URL (https://github.com/kernelci/kernelci-core.git)
+  URL of the kernelci-core repository
+KCI_CORE_BRANCH (master)
+  Name of the branch to use in the kernelci-core repository
+DOCKER_BASE
+  Dockerhub base address used for the build images
+ROOTFS_CONFIG
+  Configuration name to build RootFS images
+ROOTFS_ARCH
+  RootFS image architecture
+*/
+
+@Library('kernelci') _
+import org.kernelci.util.Job
+
+
+def listVariants(kci_core, config_list, rootfs_config, rootfs_arch) {
+    def cli_opts = ' '
+
+    if (rootfs_config) {
+            cli_opts += " --config ${rootfs_config}"
+    }
+       
+    if (rootfs_arch) {
+         cli_opts += " --arch ${rootfs_arch}"
+    }
+
+    dir(kci_core) {
+        def rootfs_config_list_raw = sh(script: """\
+./kci_rootfs \
+list_variants \
+${cli_opts} 
+""", returnStdout: true).trim()
+
+    def rootfs_config_list =  rootfs_config_list_raw.tokenize('\n')
+
+    for (String rootfs_config_raw: rootfs_config_list) {
+        def data = rootfs_config_raw.tokenize(' ')
+        def config = data[0]
+        def arch = data[1]
+        config_list.add([config, arch])
+    }
+    }
+}
+
+
+def buildRootfsStep(job, config ,arch) {
+    def node_label = "docker"
+    def parallel_builds = "4"
```

Comment:
These variabes aren't used anywhere, they seem to have been copy-pasted from `build-trigger.jpl`.